substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/SystemPropertiesSupport.java

```diff
@@ -65,7 +66,36 @@
                     "java.specification.version",
                     "java.vm.specification.name",
                     "java.vm.specification.vendor",
-                    "java.vm.specification.version"
+                    "java.vm.specification.version",
+                    "sun.arch.data.model"
+    };
+
+    /**
+     * System properties are unsupported by default.
+     */
+    private static final String[] UNSUPPORTED_PROPERTIES = {
+                    "awt.toolkit",
+                    "file.encoding.pkg",
+                    "java.awt.graphicsenv",
+                    "java.awt.printerjob",
+                    "java.home",
+                    "java.library.path",
+                    "java.runtime.name",
+                    "java.runtime.version",
+                    "java.vendor.url.bug",
+                    "java.vm.info",
+                    "sun.boot.class.path",
+                    "sun.boot.library.path",
+                    "sun.cpu.endian",
```

Comment:
that sounds like a property that can be inherited from image build time. Maybe some other too?